include/RAJA/config.hxx.in

```diff
@@ -224,6 +224,14 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 
 #define RAJA_SIMD  // TODO: Define this...
 
+#else
+
+#pragma message("RAJA_COMPILER unknown, using default empty macros.")
+
+#define RAJA_INLINE inline
```

Comment:
I think we agreed in the dev meeting that `inline` was the appropriate option for the `RAJA_INLINE` macro.